app/controllers/admin/schedules_controller.rb

```diff
@@ -23,25 +24,22 @@ def create
 
   def show; end
 
-  def edit
-    @schedules = @film.schedules
-  end
+  def edit; end
 
   def update
     if @schedule.update schedule_params
       flash[:success] = t ".schedule_update_successful"
-      redirect_to admin_film_path @film
+      redirect_to admin_film_path(@schedule.film_id) || root_url
```

Comment:
```suggestion
      redirect_to admin_film_path(@schedule.film) || root_url
```
sao phải viết như này nhỉ?